CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Change Log
 
+## Dangerzone 0.4.1 (unreleased)
+
+- Platform support: Drop Fedora 35, since it's end-of-life ([issue #308](https://github.com/freedomofpress/dangerzone/issues/308))
+- Feature: Introduce PySide6 / Qt6 support on Windows, MacOS, and Linux (dev-only) ([issue #219](https://github.com/freedomofpress/dangerzone/issues/219))
+- Feature: Adjust conversion timeouts based on the document's pages/size, and
+  allow users to disable them ([issue #327](https://github.com/freedomofpress/dangerzone/issues/327))
+- Feature: Convert documents with non-standard permissions or SELinux labels ([issue #335](https://github.com/freedomofpress/dangerzone/issues/335))
+- Feature: Add version info in the CLI and GUI ([issue #219](https://github.com/freedomofpress/dangerzone/issues/219))
+- Bug fix: Remove unused PDFtk and sudo libraries from the container image, to
+  lower its attack surface and reduce its size ([issue #232](https://github.com/freedomofpress/dangerzone/issues/232))
+- Bug fix: Report exceptions during conversions ([issue #309](https://github.com/freedomofpress/dangerzone/issues/309))
+- Bug fix: (Windows) Fix Dangerzone description on "Open With" ([issue #283](https://github.com/freedomofpress/dangerzone/issues/283))
+- Bug fix: Update Linux instructions for development on Qubes
+- Bug fix: Do not print debug logs in end-user executables
+- Bug fix: Do not leave stale temporary dirs ([issue #317](https://github.com/freedomofpress/dangerzone/issues/317))
+- Development: Improve CI stability and coverage
+  ([issue #292](https://github.com/freedomofpress/dangerzone/issues/292),
+   [issue #217](https://github.com/freedomofpress/dangerzone/issues/217),
+   [issue #229](https://github.com/freedomofpress/dangerzone/issues/229))
```

Comment:
Maybe add https://github.com/freedomofpress/dangerzone/issues/294 as well?

Reply:
Hm, good thinking. Maybe #294 warrants its own line, what do you think?

> Development: Support Dangerzone development on Fedora 37 ([issue #294](https://github.com/freedomofpress/dangerzone/issues/294))

Reply:
Fixed in cfffe5c9d5b9ecb7972c2ea9e9d79f368b06b931.